From 1cbcd6a80eb6bbea5747a5a8edc450d8fc48e976 Mon Sep 17 00:00:00 2001
From: Camelia Groza <camelia.groza@nxp.com>
Date: Thu, 19 Dec 2019 11:38:33 +0200
Subject: [PATCH] sdk_fman: decrease log severity levels

Decrease the severity of log messages from critical to appropriate
levels (warning and info).

Signed-off-by: Camelia Groza <camelia.groza@nxp.com>
---
 drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm.c      | 2 +-
 drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm_port.c | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm.c
+++ b/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm.c
@@ -2896,7 +2896,7 @@ static int __init __cold fm_load (void)
         return -ENODEV;
     }
 
-	printk(KERN_CRIT "Freescale FM module," \
+	printk(KERN_INFO "Freescale FM module," \
 		" FMD API version %d.%d.%d\n",
 		FMD_API_VERSION_MAJOR,
 		FMD_API_VERSION_MINOR,
--- a/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm_port.c
+++ b/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm_port.c
@@ -1494,11 +1494,11 @@ void LNXWRP_FM_Port_Free(void)
 static int __init __cold fm_port_load(void)
 {
 	if (LNXWRP_FM_Port_Init() != E_OK) {
-		printk(KERN_CRIT "Failed to init FM Ports wrapper!\n");
+		printk(KERN_ERR "Failed to init FM Ports wrapper!\n");
 		return -ENODEV;
 	}
 
-	printk(KERN_CRIT "Freescale FM Ports module\n");
+	printk(KERN_INFO "Freescale FM Ports module\n");
 
 	return 0;
 }
